### DISCUSSION TITLE
Showing Error

### DISCUSSION THREAD

**Discussion by tam11a** [tam11a/rayos-admin-react#17] [Q&A] [answered]
Any Component Structure for Error Message Viewer is Forms?

**Comment by asatter833** [comment, marked as answer]
const ShowError = ({ err }) => {
  const [open, setOpen] = React.useState(false);
  React.useEffect(() => setOpen(!!err), [err]);
  return (
    <Collapse
      in={open}
      sx={{
        width: "100%",
      }}
    >
      <Alert
        severity="error"
        sx={{
          width: "100%",
        }}
        onClose={() => setOpen(false)}
      >
        {err?.type === "required"
          ? "This Field is Required!"
          : err?.message || "Something wrong here!"}
      </Alert>
    </Collapse>
  );
};
